bin/css-validator

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Let's avoid using `bash` as it's not supported by Windows. We should be able to use:

```
#!/usr/bin/env node
```

`npm` wraps `bin` executables in `node` on Windows (i.e. `css-validator <filename>` would run as `node bin/css-validator.js <filename>`) and Node.js ignores `#!` on the first line of files:

https://github.com/nodejs/node/blob/v7.7.2/lib/module.js#L512-L516

// Out of time to find a link to `npm` wrapping but trust me from experience =/